packages/webpack/test/e2e-scuttle.spec.js

```diff
@@ -0,0 +1,87 @@
+const test = require('ava')
+const path = require('node:path')
+// eslint-disable-next-line ava/no-import-test-files
+const { scaffold, runScriptWithSES } = require('./scaffold.js')
+const {makeConfig} = require('./fixtures/main/webpack.config.js')
+
+const err = (intrinsic) =>
+  'LavaMoat - property "' +
+  intrinsic +
+  '" of globalThis is inaccessible under ' +
+  'scuttling mode. To learn more visit https://github.com/LavaMoat/LavaMoat/pull/360.'
```

Comment:
1. (opinion) I prefer template literals
2. Linking a PR seems maybe bad and we should either a) reference an issue instead or b)  if this is a permanent situation, link a document

Reply:
1. me too, but i further prefer keeping lines short and well indentated (am open to suggestions)
2. why an issue though? the occurrence of such an error isn't an issue to be fixed but rather something that happens intentionally which was introduced and explained in the PR. A document might make more sense, but I'm not sure it's worth focusing on atm

Reply:
nbd

Reply:
I have a stronger opinion on 2

This should be a link to our docs page or markdown file in the repo (whichever we decide is more likely to survive longer under the same URL) that explains everything